chef/cookbooks/keystone/libraries/helpers.rb

```diff
@@ -3,8 +3,11 @@ def self.service_URL(node, host, port)
     "#{node[:keystone][:api][:protocol]}://#{host}:#{port}"
   end
 
-  def self.versioned_service_URL(node, host, port)
-    service_URL(node, host, port) + '/' + node[:keystone][:api][:version] + '/'
```

Comment:
@aplanas Is there a reason, why you didn't also adjust ```service_URL``` to get rid of the ```node``` parameter there? AFAICS the only callers of ```service_URL``` are in the keystone barclamp's server recipe. (And you could change ```versioned_service_URL``` back to use it.